stories/components/login/login.styles.js

```diff
@@ -14,12 +14,8 @@ export default {
   decorators: [withCodeEditor]
 };
 
-export const darkTheme = () => html`
-  <mgt-login class="mgt-dark"></mgt-login>
-`;
-
-export const customCSSProperties = () => html`
-<mgt-login class="login"></mgt-login>
+export const customCssProperties = () => html`
```

Comment:
This is a case sensitive name at the moment in the codeAddOn
```suggestion
export const customCSSProperties = () => html`
```